cmd/client/command/profile.go

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (c) 2017, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package command
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/spf13/cobra"
+)
+
+// ProfileCmd defines member command.
+func ProfileCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "profile",
+		Short: "Start and stop CPU and memory profilers",
+	}
+
+	cmd.AddCommand(infoProfileCmd())
+	cmd.AddCommand(startProfilingCmd())
+	cmd.AddCommand(stopProfilingCmd())
+	return cmd
+}
+
+func infoProfileCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "info",
+		Short: "Show memory and CPU profile file paths.",
```

Comment:
```suggestion
		Short: "Show memory and CPU profile file paths",
```

Consistent with other commends, no period in short usage. So are others